src/api/Api.js

```diff
@@ -26,7 +26,7 @@ export default /* @ngInject */ function ($http, $q) {
   angular.forEach(['get', 'put', 'post', 'delete'], (operationType) => {
     self[operationType] = (url, opt) => {
       const options = angular.extend({}, opt);
-      options.method = angular.uppercase(operationType);
+      options.method = operationType.toUpperCase();
```

Comment:
Watch out, `angular.uppercase(null)` may return `null`, wether `null. toUpperCase ()` will throw an error ! You can `(operationType || '')toUpperCase()` instead

Reply:
yes sure but i think it's safe here : `angular.forEach(['get', 'put', 'post', 'delete'], (operationType) => ...`

Reply:
Indeed, I was too lazy to check :D

Reply:
In that case `self[operationType]` is equivalent to `self[null]`